refactor: Slash commands (#936)
discordjs/guide: guide/interactions/slash-command-permissions.md

Next, we define an array of objects. Each object needs to have two properties: `id` and `permissions`. The `id` should be the command's id and the `permissions` should be an array of [Application Command Permissions Structure](https://discord.com/developers/docs/interactions/application-commands#application-command-permissions-object-application-command-permissions-structure) objects. 

This `Application Command Permissions Structure` object needs to have three properties: `id`, `type`, and `permission`. The `id` will be the user id or role id that you want to allow running the command. The `type` specifies what type of `id` it is. Type `1` is for roles, and type `2` is for users. Finally, the `permission` specifies if you want to allow or disallow this `id` to use the command. If `permission` is `true` then it means allow, if it is `false` then it means disallow. Since we want to allow users, we set `permission` to true.

> Finally, we make a `PUT` request to update command overwrites. We use the `Routes.guildApplicationCommandsPermissions()` route, and pass our `fullPermissions` array as the body of the `PUT` request.

Reviewer: ```suggestion
Finally, you can make a `PUT` request to update command overwrites. Using the `Routes.guildApplicationCommandsPermissions()` route and passing your `fullPermissions` array as the body of the `PUT` request.
```

> We can now run the deployment script again:

Reviewer: ```suggestion
You can now run the deployment script again:
```
